tasks/section3.yml

```diff
@@ -331,15 +398,19 @@
       - rule_3.4.4
 
 - name: "SCORED | 3.4.4 | PATCH | Ensure permissions on /etc/hosts.allow are configured"
-  command: /bin/true
+  file:
+      dest=/etc/hosts.allow
+      owner=root
+      group=root
+      mode=0644
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md